### PR TITLE
[zh] Rework Pod Lifecycle page

### PR DESCRIPTION
**Pull request by tengqm**
A lot of things have changed upstream.

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -1,180 +1,829 @@
...
+任何给定的 Pod （由 UID 定义）从不会被“重新调度（rescheduled）”到不同的节点；
+相反，这一 Pod 可以被一个新的、几乎完全相同的 Pod 替换掉。
+如果需要，新 Pod 的名字可以不变，但是其 UID 会不同。
+
+如果某物声称其生命期与某 Pod 相同，例如存储{{< glossary_tooltip term_id="volume" text="卷" >}}，
+这就意味着该对象在此 Pod （UID 亦相同）存在期间也一直存在。
+如果 Pod 因为任何原因被删除，甚至某完全相同的替代 Pod 被创建时，
+这个相关的对象（例如这里的卷）也会被删除并重建。
+
+{{< figure src="/images/docs/pod.svg" title="Pod diagram" width="50%" >}}
```
Comment (sftim) [inline content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:110]: If you want, you can translate the title too.

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -1,180 +1,829 @@
...
+`status`             | Indicates whether that condition is applicable, with possible values "`True`", "`False`", or "`Unknown`".
+`lastProbeTime`      | Timestamp of when the Pod condition was last probed.
+`lastTransitionTime` | Timestamp for when the Pod last transitioned from one status to another.
+`reason`             | Machine-readable, UpperCamelCase text indicating the reason for the condition's last transition.
+`message`            | Human-readable message indicating details about the last status transition.
+-->
+字段名称             | 描述
+:--------------------|:-----------
+`type`               | Pod 状况的名称
+`status`             | 表明该状况是否适用，可能的取值有 "`True`", "`False`", or "`Unknown`"
```
Comment (sftim) [inline content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:301]: I'd expect to see “`True`, `False` or `Unknown`” translated here - it's not the logical OR operation.

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 3]
```diff
@@ -1,180 +1,829 @@
...
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+<!--
+Startup probes are useful for Pods that have containers that take a long time to
+come into service. Rather than set a long liveness interval, you can configure
+a separate configuration for probing the container as it starts up, allowing
+a time longer than the liveness interval would allow.
+-->
+对于所包含的容器需要较长时间待能启动就绪的 Pod 而言，启动探针是有用的。
```
Comment (SataQiu) [inline content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:580]:
```suggestion
对于所包含的容器需要较长时间才能启动就绪的 Pod 而言，启动探针是有用的。
```

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 4]
```diff
@@ -1,180 +1,829 @@
...
+      {{< note >}}
+      如果 `preStop` 回调所需要的时间长于默认的体面终止限期，你必须修改
+      `terminationGracePeriodSeconds` 属性值来使其正常工作。
+      {{< /note >}}
+
+   1. `kubelet` 接下来触发容器运行时发送 TERM 信号给每个容器中的进程 1。
+
+      {{< note >}}
+      Pod 中的容器会在不同时刻收到 TERM 信号，接收顺序也是不确定的。
+      如果关闭的顺序很重要，可以考虑中使用 `preStop` 回调逻辑来协调。
```
Comment (SataQiu) [inline content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:689]:
```suggestion
      如果关闭的顺序很重要，可以考虑使用 `preStop` 回调逻辑来协调。
```

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:110]: good catch.

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:301]: True, False, Unknown are the verbal values shown in `kubectl` output. It is not supposed to be translated.
However, the 'or' should be translated.
Fixed.

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md (hunk 3 above)
Comment (tengqm) [reply content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:580]: Good catch. Fixed.

File: content/zh/docs/concepts/workloads/pods/pod-lifecycle.md (hunk 4 above)
Comment (tengqm) [reply content/zh/docs/concepts/workloads/pods/pod-lifecycle.md:689]: Fixed.

**Comment by tengqm** [comment]
@sftim Thanks for the feedback on restart-policy. I have 100% sure I translated that paragraph, but I am confused where the translation has gone ... 
Anyway, I have reworked that section.

**Comment by sftim** [comment]
I'll leave further reviews to people who are fluent in Chinese.

**Pull request merged**